server/src/main/java/org/jboss/as/server/BootstrapListener.java

```diff
@@ -68,7 +69,7 @@ public StabilityMonitor getStabilityMonitor() {
         return monitor;
     }
 
-    public void printBootStatistics() {
+    public void printBootStatistics(boolean isHostController, boolean isMaster, List<String> configFiles) {
```

Comment:
Remove the `boolean isMaster`, you don't need it, you can infer the behavior by using `configFiles.size() == 2`, (configFiles.size() == 2, then you are in a master DC)